### PR TITLE
dkms: add module load parameters

### PR DESCRIPTION
**Pull request by wangyiyanga**
nuclear_pageflip=1 enable_guc=0x7 max_vfs=7 modeset=1 fastboot=1

### CODE REVIEW

**Review by xyzhao2018 — CHANGES_REQUESTED** [review]
Please add "Test done" part when ready.

**Review by feijiang1 — APPROVED** [review]
LGTM

**Review by xyzhao2018 — COMMENTED** [review]
Please add "Test done" part, descript the simple test flow and result.

File: kernel/module/main.c [hunk 1]
```diff
@@ -2739,7 +2739,7 @@ static int unknown_module_param_cb(char *param, char *val, const char *modname,
  * zero, and we rely on this for optional sections.
  */
 static int load_module(struct load_info *info, const char __user *uargs,
-		       int flags, bool no_args)
```
Comment (xyzhao2018) [inline kernel/module/main.c:2742]: As load_moudle is a common API in kernel/module/main.c, if update the parameter format, will it impact other load module process?

Comment (wangyiyanga) [reply kernel/module/main.c:2742]: no, 
it's only extending for dGPU driver，
legacy working no changes.

**Review by xyzhao2018 — APPROVED** [review]

**Pull request merged**